pkg/prometheus/promcfg.go

```diff
@@ -610,7 +608,7 @@ func generateRemoteWriteConfig(version semver.Version, specs []v1.RemoteWriteSpe
 		}
 
 		if spec.WriteRelabelConfigs != nil {
-			relabelings := []yaml.MapSlice{}
+			var relabelings []yaml.MapSlice
```

Comment:
Actually looking at this again, I remember why we did this, consistently everywhere. This way we make sure we actually have a slice whereas with `var ...`, we may have nil, which can cause curious situations. I prefer actually initializing the empty slice.